internal/controlplane/olm_cleanup.go

```
@@ -0,0 +1,339 @@
+		if strings.HasPrefix(list.GroupVersion, "operators.coreos.com/") {
+			return true
+		}
+	}
+	return false
+}
+
+// removeOrphanedOLMMetadata finds all resources owned by orphaned child
+// operator CSVs using the olm.owner label and strips OLM ownerReferences
+// and labels from them. This works for any resource type OLM manages —
+// no hardcoded resource list needed.
+func (c *OLMCleaner) removeOrphanedOLMMetadata(ctx context.Context) error {
```

golangci-lint (2.12.2)

[error] (*OLMCleaner).removeOrphanedOLMMetadata - result 0 (error) is always nil

(unparam)